README.md

```diff
@@ -69,7 +69,8 @@ and if you plan to run the emulator with Docker you must use the environment var
 | `--skip-tx-validation` | `FLOW_SKIPTRANSACTIONVALIDATION` | `false`        | Skip verification of transaction signatures and sequence numbers                                                                                                                                                                                    |
 | `--host`                      | `FLOW_HOST`                  | ` `            | Host to listen on for emulator GRPC/REST/Admin servers (default: All Interfaces)                                                                                                                                                                                              |
 | `--chain-id`                  | `FLOW_CHAINID`               | `emulator`     | Chain to simulate, if 'mainnet' or 'testnet' values are used, you will be able to run transactions against that network and a local fork will be created..  Valid values are: 'emulator', 'testnet', 'mainnet'                                       |
-| `-redis-url`                  | `FLOW_REDIS_URL`             | ''             | Redis-server URL for persisting redis storage backend ( `redis://[[username:]password@]host[:port][/database]` )                                                                                                                                    |
+| `--redis-url`                 | `FLOW_REDIS_URL`             | ''             | Redis-server URL for persisting redis storage backend ( `redis://[[username:]password@]host[:port][/database]` )                                                                                                                                    |
+| `--start-block-height`        | `FLOW_STARTBLOCKHEIGHT`             | `0`             | Start block height to use when starting the network using 'testnet' or 'mainnet' as the chain-id    |
```

Comment:
not sure if this is the correct envvar name. I couldn't find them defined anywhere

Reply:
@peterargue  I feel like it should be `FLOW_START_BLOCK_HEIGHT` 

it should be in "https://github.com/psiemens/sconfig/blob/893e07c4e321243de5ca6596cdb74fbe9046e322/sconfig.go#L162" 

default replace of viper is "-" to "_" afaik 

I will check all flags and go all over them